packages/core/src/sync-realtime/service.test.ts

```diff
@@ -112,7 +112,7 @@ test("start() sync realtime data with batch method", async (context) => {
   await service.setup();
   service.start();
 
-  await testClient.mine({ blocks: 8 });
+  await testClient.mine({ blocks: 10 });
```

Comment:
Why did you have to make changes in this file?

Reply:
Because we changed the behavior of poll to run the "process" function immediately instead of wait the interval to start processing.